test/alternative_spec.rb

```diff
@@ -0,0 +1,57 @@
+require_relative "../src/lib/update-alternatives/model/alternative"
+require_relative "../src/lib/update-alternatives/model/choice"
+require "cheetah"
+
+describe "An alternative" do
```

Comment:
please see some examples and howto for rspec. It have own specific structure. E.g. like http://betterspecs.org/ or http://blog.teamtreehouse.com/an-introduction-to-rspec ( second is a bit old, but idioms do not change )